aten/src/ATen/native/cuda/BatchLinearAlgebraLib.cu

```diff
@@ -826,6 +826,55 @@ Tensor& orgqr_helper_cusolver(Tensor& result, const Tensor& tau) {
   return result;
 }
 
+void apply_lu_cusolver_looped(Tensor& self, Tensor& pivots, Tensor& infos, bool get_pivots) {
+  auto infos_data = infos.data_ptr<int>();
+
+  AT_DISPATCH_FLOATING_AND_COMPLEX_TYPES(self.scalar_type(), "lu_cusolver", [&]{
+    auto self_data = self.data_ptr<scalar_t>();
+    auto self_stride = matrixStride(self);
+    int batch_size = cuda_int_cast(batchCount(self), "batch size");
```

Comment:
batch_size should be int64_t, cuda_int_cast is not needed here because we don't pass the batch_size to cuSOLVER routine. It's only batched cuBLAS requires int type. 
```suggestion
    auto batch_size = batchCount(self);
```